src/components/Floating/FloatingButton.tsx

```diff
@@ -0,0 +1,92 @@
+'use client';
+
+import { useState } from 'react';
+import Image from 'next/image';
+import { twMerge } from 'tailwind-merge';
+import { ChatBot } from '../ChatBot/ChatBot';
+
+type FloatingButtonProps = {
+  type: 'top' | 'ask';
+};
+
+export function FloatingButton({ type }: FloatingButtonProps) {
+  const [isOpen, setIsOpen] = useState(false);
+  const [isHovered, setIsHovered] = useState(false);
+
+  const handleClick = () => {
+    if (type === 'top') {
+      if (typeof window !== 'undefined') {
+        window.scrollTo({ top: 0, behavior: 'smooth' });
+      }
+    } else if (type === 'ask') {
+      setIsOpen(prev => !prev);
+    }
+  };
+
+  const icons = {
+    top: {
+      default: '/icons/floatVector.svg',
+      hovered: '/icons/floatVectorwhite.svg',
+      width: 32,
+      height: 32,
+    },
+    ask: {
+      default: '/icons/floatChat.svg',
+      hovered: '/icons/floatChatwhite.svg',
+      width: 35,
+      height: 35,
+    },
+  };
+
+  return (
+    <>
+      <button
+        type="button"
+        onClick={handleClick}
+        onMouseEnter={() => setIsHovered(true)}
+        onMouseLeave={() => setIsHovered(false)}
```

Comment:
```suggestion
        onMouseEnter={() => setIsHovered(true)}
        onMouseLeave={() => setIsHovered(false)}
```

호버는 css로만 처리해줘도 좋을거 같습니다
className에 `hover:호버시적용할css`